### DISCUSSION TITLE
[Question]: Is it possible to get more debug about why an assistant action API call failed?

### DISCUSSION THREAD

**Discussion by dividor** [danny-avila/LibreChat#2403] [Q&A]
### What is your question?

Is it possible to get more debug to confirm the URL called, headers etc, so see why the API call failed when run in an assistant action?

### More Details

I am working with assistants and have configured some actions to call an external API. The API is available and the openAPI spec I have loaded seems to be ok, and a curl command returns data. However, when the assistant 'Talks to' the API, I see the payload mentioned correctly, but the API call fails. Looking in the debug and error logs, all I have is ...

`LibreChat         | 2024-04-13 19:50:35 error: API call to <HOST> failed Cannot read properties of undefined (reading 'status')`

### What is the main subject of your question?

Other

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This kind of error happens when a connection could not be established. Also, check the debug logs in the project directory, `./logs`
